ppr-ui/src/components/common/AccountInfo.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div id="account-info">
+    <h4 class="header mb-5">
+      {{ title }}
+      <v-tooltip
+        if="tooltipContent"
+        top
+        nudge-right="3"
+        content-class="top-tooltip pa-5"
+        transition="fade-transition"
+        data-test-id="submitting-party-tooltip"
+      >
+        <template v-slot:activator="{ on }">
+          <v-icon class="mt-n1" color="primary" v-on="on">
+            mdi-information-outline
+          </v-icon>
+        </template>
+        {{ tooltipContent }}
+      </v-tooltip>
+    </h4>
+    <v-card flat class="rounded">
+      <v-simple-table v-if="accountInfo" data-test-id="account-info-table">
+        <template v-slot:default>
+          <thead>
+            <tr>
+              <th class="pl-6 py-4">
+                Name
+              </th>
+              <th class="py-4">
+                Mailing Address
+              </th>
+              <th class="py-4">
+                Email Address
+              </th>
+              <th class="py-4">
+                Phone Number
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr class="table-info">
+              <td class="account-name pl-6 py-6">
+                <v-icon>
+                  {{ accountInfo.isBusinessAccount ? 'mdi-domain' : 'mdi-account' }}
+                </v-icon>
+                <span class="pt-1 font-weight-bold">
+                  {{ accountInfo.name }}
+                </span>
+              </td>
+              <td class="py-6">
+                <BaseAddress
+                  id="submitting-party-address"
+                  :schema="PartyAddressSchema"
+                  :value="accountInfo.mailingAddress"
+                />
+              </td>
+              <td class="py-6">
+                {{ accountInfo.accountAdmin.email }}
+              </td>
+              <td class="py-6">
+                {{ toDisplayPhone(accountInfo.accountAdmin.phone) }}
+                <span v-if="accountInfo.accountAdmin.phoneExtension">
+                  Ext {{ accountInfo.accountAdmin.phoneExtension }}
+                </span>
+              </td>
+            </tr>
```

Comment:
Not a huge deal but there is a lot of tags here that could very easily fit on 1 line. 
I guess it's preference but the condensing might make for better readability. Your call though. 
ie `<th class="pl-6 py-4">Name</th>` as example, i'm not sure its easier to read as 3 lines vs 1.

Reply:
Formatted by Prettier 🤷🏻 I want to review its config once I get to it sometime later.

Reply:
haha all good. Like i said, this is just preference. 
Up to you!